src/locales/en.json

```diff
@@ -26,7 +26,8 @@
       "graph": "Graph",
       "no-node": "No Node Selected",
       "index": "Index",
-      "label": "Label"
+      "label": "Label",
```

Comment:
can you please make a root section follows component path "components/CodeEditor/MonacoEditor" and put your settings there as I cant see how these relate to Projects.  Seems that Projects section is a dumping ground for things.